src/main/java/com/github/fge/jsonpatch/JsonPatchFactoryUtil.java

```diff
@@ -0,0 +1,55 @@
+package com.github.fge.jsonpatch;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utilities for JsonPatchFactory. It provides a default implementation of JsonPatchFactory that
+ * has all the RFC6902 operations registered.
+ */
+public class JsonPatchFactoryUtil {
+    private static Map<String, Class<? extends JsonPatchOperation>> DEFAULT_OPERATIONS =
+        Collections.unmodifiableMap(
+            new HashMap<String, Class<? extends JsonPatchOperation>>() {{
```

Comment:
ignore it, it isn't an issue I was just curious because I hadn't looked into double brace init.